New Import/Export Instructions with TOC update. (#1464)
MicrosoftDocs/SCCMdocs: sccm/apps/deploy-use/import-export-applications.md

---

> 

Reviewer: Here's how the metadata block should look, starting at line 1 of the article:

```
---
title: Import and export applications
titleSuffix: Configuration Manager
description: Learn how to import and export applications in Configuration Manager to share between separate hierarchies.
ms.date: 05/23/2019
ms.prod: configuration-manager
ms.technology: configmgr-app
ms.topic: conceptual
ms.assetid: dba00e54-9d5b-4f6b-916d-ead48c66e288
author: aczechowski
ms.author: aaroncz
manager: dougeby
ms.collection: M365-identity-device-management
---
```

- I'm hoping if you set the author field to me, the bot will let you reopen this PR 
- The asset ID is just a random GUID that I generated via New-Guid cmdlet 😉

Author: Fixed in #1585

# Mandatory fields.
title: Import/Export Applications
titleSuffix: Configuration Manager
description: Import and export applications.
author: chriskibble
ms.author: NO VALUE SET # Microsoft employees only
ms.date: 5/6/2019
ms.topic: conceptual
ms.technology: configmgr-app
---

> # Create applications in Configuration Manager

Reviewer: change this to `Import and export applications`

*Applies to: System Center Configuration Manager (Current Branch)*

> Configuration Manager allows for the import and export of applications within the console.  This allows you to copy applications in to other environments, such as copying from a test environment into a production environment.

Reviewer: A suggested rewording:

```suggestion
Use Configuration Manager to import and export applications to another hierarchy. For example, copy an application from a test environment to a production environment.
```

Author: Made some minor changes to your suggestion and submitted as #1585

> ## <a name="bkmk_export"></a> Export an Application

Reviewer: Use sentence case, and the anchor isn't always necessary. Actually, since this article is in the context of apps, then I think the "an application" part is redundant. So this header can be really simplified to just:

```suggestion
## Export
```

Author: Fixed in #1585

> 1. Within the Configuration Manager console, right click on the application to export and select **Export**.

Reviewer: Avoid use of the verb "click" as it's not inclusive (it implies the ability to use a mouse button). For more information, see [click in the style guide](https://docs.microsoft.com/style-guide/a-z-word-list-term-collections/c/click).

```suggestion
1. In the Configuration Manager console, select the application to export. In the ribbon, choose **Export**.
```

Author: Fixed in #1585

> 1. On the **General** screen, enter a path to a new ZIP file to export into.  Optionally define if you'd like to export *dependencies, supersedence relationships, conditions, and virtual environments* and also if you'd like to export *content for the selected applications and dependencies*.  Enter any administrator comments, if desired, and click **Next**.

Reviewer: Same note as above in the next few lines regarding use of "click". I generally use "select" unless it gets repetitive or just sounds better to use "choose"

Author: Fixed in #1585

1. Verify the application and any dependencies are listed on the **Related Objects** page and click **Next**.
1. On the Summary page, click **Next**.

> 1. Once the process is complete, the ZIP file will be created and you can click the **Close** button.

Reviewer: "will be created" is weird tense that can be hard to localize in other languages. I suggest:

```suggestion
1. Once the process completes, it creates the ZIP file, and you can close the wizard.
```

Author: Fixed in #1585

> [!IMPORTANT]
> If you're going to copy this to another environment, you'll need to take both the created ZIP file and the folder that accompanies it.  The ZIP file must exist in the same directory as the created folder.

> ## <a name="bkmk_import"></a> Import Export an Application

Reviewer: Similar as above, simplify this:

```suggestion
## Import
```

Author: Fixed in #1585

> [!NOTE]

> > Applications can only be imported from UNC paths, you will not be able to import from your local disk directly.

Reviewer: I generally avoid use of "will" and passive voice like "be imported" - make it more active. I suggest:

```suggestion
> You can only import applications from UNC paths, you can't directly import from your local disk.
```

Author: Fixed in #1585

> 1. Within the Configuration Manager console, right click on the application node in the left margin  **Import Application**.

Reviewer: Avoid the use of "click" or "right-click" if possible. In this example, you can say:

```suggestion
1. In the Configuration Manager console, select the **Applications** node. In the Create group of the ribbon, choose **Import Application**.
```

Author: Fixed in #1585

1. Select the ZIP file that you'd like to import and click **Next**.

> 1. The File Content window will show what will happen with the application you're importing.  Click **Next**.

Reviewer: ```suggestion
1. The File Content window shows what what will happen when you import the application. Select **Next**.
```
(In this case, "will happen" is appropriate because it's something happening in the future.)

Author: Fixed in #1585

1. Review the summary screen and click **Next**.

> 1. On the Completion screen, click the **Close** button.  Your application has now been imported.

Reviewer: ```suggestion
1. Close the wizard. The application is now available in the site.
```

Author: Fixed in #1585

> ## <a name="bkmk_whatsnext"></a> What's Next

Reviewer: ```suggestion
## See also
```

Author: Fixed in #1585

 
Automate the import and export of applications using PowerShell.

> * [Import-CMApplication](https://docs.microsoft.com/en-us/powershell/module/configurationmanager/import-cmapplication)

Reviewer: Remove the locale from the URL 😄 

But I ❤️ that you added this!

Author: Fixed in #1585 ... and thanks ❤️
